pkgs/top-level/all-packages.nix

```diff
@@ -2473,7 +2473,8 @@ in
     cudatoolkit_9_1
     cudatoolkit_9_2
     cudatoolkit_10
-    cudatoolkit_10_0;
+    cudatoolkit_10_0
+    cudatoolkit_10_1;
 
   cudatoolkit = cudatoolkit_9;
```

Comment:
Might be time to bump that one to cudatoolkit_10

Reply:
done